galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 description = 'Galasa CICS/TS Manager'
 
-version = '0.28.0'
```

Comment:
needs to be put back to 0.28.0 as this is the inflight release.  However well done for remembering to do it - I usually forget

Reply:
Will do 👍